libs/core/tests/unit_tests/fake/test_fake_chat_model.py

```diff
@@ -8,30 +8,31 @@
 from langchain_core.messages import AIMessage, AIMessageChunk, BaseMessage
 from langchain_core.messages.human import HumanMessage
 from langchain_core.outputs import ChatGenerationChunk, GenerationChunk
+from tests.unit_tests.stubs import AnyStr
 
 
 def test_generic_fake_chat_model_invoke() -> None:
     # Will alternate between responding with hello and goodbye
     infinite_cycle = cycle([AIMessage(content="hello"), AIMessage(content="goodbye")])
     model = GenericFakeChatModel(messages=infinite_cycle)
     response = model.invoke("meow")
-    assert response == AIMessage(content="hello")
+    assert response == AIMessage(content="hello", id=AnyStr())
```

Comment:
instead of `AnyStr` should we just switch to asserting e.g. `response.content = "hello"`

Reply:
Or have a helper method of `_message_equal_ignore_id(a, b)`

`AnyStr` feels more likely to confuse